jhub_apps/service/routes.py

```diff
@@ -27,7 +27,8 @@
     AuthorizationError,
     HubApiError,
     ServerCreation,
-    User,
+    User, Repository,
```

Comment:
Is there any Python linter that we follow? This looks out of style here

```suggestion
    User,
    Repository,
```

Reply:
We do use `ruff`, I noticed it didn't do that several times, might be a config or setting we're missing.